site/src/api/queries/templates.ts

```diff
@@ -236,6 +237,38 @@ export const resources = (versionId: string) => {
   };
 };
 
+export const templateFiles = (fileId: string) => {
+  return {
+    queryKey: ["templateFiles", fileId],
```

Comment:
Do you think it makes sense to update the query keys for `templateFiles` and `previousTemplateVersion` to start with something like `"template"`? Feels like it'd be a good way to make sure we have an easy way to invalidate any template queries down the line

Reply:
If we had the template ID or name I would say yes, but since they are "isolated" I'm ok to keep the current names until we see value in changing them.